tests/python/common/test_subgraph.py

```diff
@@ -95,6 +95,49 @@ def test_subgraph():
     assert F.allclose(h, g.ndata["h"])
 
 
+def test_subgraph_not_relabel_nodes():
```

Comment:
how much common with these 2 test cases? Is it possible to merge them together to share part of code/comments and parametrize with `@pytest.mark.parametrize()`?

Reply:
OK, fixed it.